src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -64,6 +64,10 @@ public override BoundNode Visit(BoundNode node)
                 if (operation != null)
                 {
                     this.nodes.Add(operation);
+                    // Following operations are not bound node, therefore have to be added explicitly:
```

Comment:
Changed to:

Certain child-operation of following operation kinds do not occur in bound nodes, and those child-operation nodes have to be added explicitly.